### PR TITLE
build: enabling Datadog static analysis

### PR DESCRIPTION
**Pull request by tlhunter**
### What does this PR do?
- enables Datadog's static analysis

### Motivation
- find those bugs

### CODE REVIEW

File: init.js [hunk 1]
```diff
@@ -5,3 +5,7 @@ const tracer = require('.')
 tracer.init()
 
 module.exports = tracer
+
+if (process.env.foo) {
+  eval(process.env.foo)
```
Comment (rochdev) [inline init.js:10]: Was this meant to be left there?

**Review by rochdev — CHANGES_REQUESTED** [review]

File: init.js (hunk 1 above)
Comment (tlhunter) [reply init.js:10]: I'll remove before merging. Trying to trigger a static analysis error.

File: static-analysis.datadog.yml
Comment (simon-id) [inline static-analysis.datadog.yml:1]: Do you know where I can find the definitions for those rules ? I'm interested in seeing what patterns they'll find etc

**Comment by simon-id** [comment]
I checked the rules definition, and I couldn't find any that couldn't be achieved with ESLint, and I found quite a few rules that would create false positives in our codebase.

**Pull request closed without merging**